packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4994,7 +5075,7 @@ class Row extends Flex {
 ///    use a [Column] inside a scrolling container.
 ///  * [Spacer], a widget that takes up space proportional to its flex value.
 ///  * The [catalog of layout widgets](https://flutter.dev/widgets/layout/).
-class Column extends Flex {
+class Column extends StatelessWidget with MultiChildRenderObjectWidgetMixin, FlexMixin {
```

Comment:
Why MultiChildRenderObjectWidgetMixin is needed?

Reply:
Removed